unity-renderer/Assets/Plugins/JSFunctions.jslib

```diff
@@ -36,6 +36,21 @@ mergeInto(LibraryManager.library, {
   },
   CheckURLParam: function(targetParam) {    
     return window.DCL.queryString.has(Pointer_stringify(targetParam))
+  },
+  GetURLParam: function(targetParam) {
+    const stringifiedTargetParam = Pointer_stringify(targetParam)
+    
+    if(window.DCL.queryString.has(stringifiedTargetParam)) {
```

Comment:
what is DCL.queryString?

Can we use `new URL(document.location)` instead?

Reply:
May I suggest to send this information via KernelConfig instead? This file should not be touched actually

Reply:
I removed them in [this commit](https://github.com/decentraland/unity-renderer/pull/2345/commits/da6dc59b49b8fdbeb15cf789a097d3df67f4eef2) and [created this issue](https://github.com/decentraland/unity-renderer/issues/2519) for the future